README.md

```diff
@@ -126,8 +126,13 @@ These composites are also the motivation for the short aliases of payloads, as t
 For more examples, please see the [sample command](https://github.com/EluciusFTW/SpectreCoff/blob/main/src/spectrecoff-cli/commands/Output.fs).
 
 ### Modules
+This Bar Chart (created using SpectreCoff) breaks down our progress taking over the Spectre.Console modules.
+Much is already usable, but there is still a way to go. 
+
+![](assets/progress.png)
```

Comment:
It looks a bit big-ass like this, try playing with the percentages, something like:

`<img align="right" src="assets/progress.png" width="40%" />`

Reply:
Didn't even know you can include html there, nice!